models/user.ts

```diff
@@ -0,0 +1,16 @@
+import { Schema, model } from "mongoose";
+
+const userSchema = new Schema(
+  {
+    userID: Number,
+    fname: String,
+    lname: String,
+    createdAt: Date,
+    updatedAt: Date,
+  },
+  { timestamps: true }
+);
+
+const userModel = model("week1_demos", userSchema);
```

Comment:
Use a better name for your collection that matches the entity you want to store, i.e `User`.